TestResources_v3/Virtual Time/VirtualTimers.swift

```diff
@@ -0,0 +1,31 @@
+//
+// VirtualTimers.swift
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+@testable import StreamChatClient_v3
+
+struct VirtualTimeTimer: StreamChatClient_v3.Timer {
```

Comment:
Why isn't this entity inside `VirtualTime` as its default timer?

Reply:
We already talked about this https://github.com/GetStream/stream-chat-swift/pull/268#discussion_r425797226